libbuteosyncfw/profile/SyncProfile.cpp

```diff
@@ -558,7 +558,7 @@ quint32 SyncProfile::syncOnChangeAfter() const
         }
     }
     //}
```

Comment:
a commented out closing brace is a bit scary

Reply:
The opening brace was commented out as well, seems like old code that could have been just removed instead. I've removed these since they don't do anything.